lib/simulation/openfisca/mapping/index.js

```diff
@@ -16,31 +16,26 @@ var isNotValidValue = function(value) {
 var getPeriods = function(dateDeValeur) {
     dateDeValeur = moment(dateDeValeur).freeze();
     return {
-        moisCourant: {
+        thisMonth: {
             key: dateDeValeur.format('YYYY-MM')
```

Comment:
Is there any reason to wrap all these periods in objects with a single key-value pair?! Why not simply store the value?